api/taskcluster/webhook.go

```diff
@@ -163,12 +163,18 @@ func GetCheckSuiteEventInfo(checkSuite github.CheckSuiteEvent, log shared.Logger
 			group.TaskGroupID = taskID
 		}
 
-		log.Debugf("Adding task: %s, id: %s, status: %s", run.GetName(), taskID, run.GetStatus())
+		log.Debugf("Adding task: %s, id: %s, conclusion: %s", run.GetName(), taskID, run.GetConclusion())
+
+		// Match the GitHub Check Runs API to the old Taskcluster status API.
+		state := "completed"
+		if run.GetConclusion() != "success" {
+			state = run.GetConclusion()
+		}
```

Comment:
Nit (though I know `GetConclusion` is cheap)

```suggestion
		state := run.GetConclusion()
		if state == "success" {
			// Checked in ExtractArtifactURLs.
			state = "completed"
		}
```